Core/SConscript

```diff
@@ -16,21 +16,28 @@ def recursive_glob(treeroot, pattern):
 	return results
 # 	('SoarKernel/SoarKernel.cxx',       recursive_glob(kernel_env.Dir('SoarKernel/src').abspath(), '*.cpp')),    # Kernel
 
-def CheckSWIG(env):
+def SWIGVersion(env):
 	if not env.WhereIs('swig'):
-		return False
-
+		return None
 	p = sub.Popen('swig -version'.split(), stdout=sub.PIPE)
 	out = p.communicate()[0].decode().split()
 	p.wait()
 
-	version = tuple(int(x) for x in out[2].split('.'))
+	return tuple(int(x) for x in out[2].split('.'))
+
+
+def CheckSWIG(env):
+	version = SWIGVersion(env)
+	if version is None:
+		return False
+
 	if version >= (1, 3, 31):
 		return True
 
 	print('swig version 1.3.31 or higher is required')
```

Comment:
Funny I hadn't actually noticed this version check before. 1.3.31 seems like a crazy low version.

Reply:
4.2.0 in comparison is a crazy high version, which is why I opted to make abi3 building completely optional and opt-in via environment variable, because else we'll probably be breaking a bunch of workflows.

Reply:
Aaah, that makes sense. Could you add a comment somewhere about that?